src/common/utils/MetaKeyUtils.h

```diff
@@ -419,6 +419,56 @@ class MetaKeyUtils final {
   static std::string diskPartsVal(const meta::cpp2::PartitionList& partList);
 
   static meta::cpp2::PartitionList parseDiskPartsVal(const folly::StringPiece& rawData);
+
+  // job related
+  static const std::string& jobPrefix();
+
+  static std::string jobPrefix(GraphSpaceID spaceId);
+
+  /**
+   * @brief Encoded job key, it should be
+   * kJobTable+spceId+jobid
+   */
+  static std::string jobKey(GraphSpaceID spaceId, JobID iJob);
+
+  static bool isJobKey(const folly::StringPiece& rawKey);
+
+  /**
+   * @brief Decode val from kvstore, return
```

Comment:
Encode?

Reply:
Haha, I moved this function from another file, I fixed it